src/spi.rs

```diff
@@ -768,18 +874,15 @@ where
         &mut self,
         operations: &mut [embedded_hal_0_2::blocking::spi::Operation<'_, u8>],
     ) -> Result<(), Self::Error> {
-        self.transaction(|bus| {
-            for operation in operations {
-                match operation {
-                    embedded_hal_0_2::blocking::spi::Operation::Write(write) => bus.write(write),
-                    embedded_hal_0_2::blocking::spi::Operation::Transfer(words) => {
-                        bus.transfer_in_place(words)
-                    }
-                }?;
+        for op in operations {
```

Comment:
This still needs work. Should just convert the 0.2 Operation to 1.0 Operation, otherwise it will not do it in one transaction.